apps/docs/components/AnimatedGradientBox/AnimatedGradientBox.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
```

Comment:
```suggestion
```

Reply:
This is a limitation of the ESLint pkugin enforcing the header comments...